assets/src/edit-story/output/test/constants.js

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Need to move this to a folder called `testUtils` or `_utils`, otherwise Jest is looking for some tests in this file (and can't find them)

Reply:
Whoops, yeah, slipped my mind!!